02-peer-protocol.md

```diff
@@ -377,6 +383,12 @@ The recipient:
   - on receipt of a valid `funding_signed`:
     - SHOULD broadcast the funding transaction.
 
+#### Rationale
+
+We decide on `option_static_remotekey` at this point when we first have to generate the commitment
+transaction.  Even if a later reconnection does not negotiate this parameter, this channel will honor it.
```

Comment:
What does the second sentence mean? That this is actually a _connection_ level value, rather than local to this specific channel negotiation?

Reply:
You can't turn it off by reconnecting without the feature later.  If someone downgrades, it's undefined.